src/Teams.Web/Controllers/ManageSprintsController.cs

```diff
@@ -116,13 +119,13 @@ public async Task<IActionResult> CompleteTaskInSprint(int taskId, bool isComplet
 
             if (sprint == null)
             {
```

Comment:
Please, decide, use brackets for one-line code after, or not.
From time to time it might happen to decide to use brackets if always not use them, up to situation it will be more readable.
But not in this case, here we have same situations and different approach.